app/routes/joblistings/components/JoblistingDetail.js

```diff
@@ -143,27 +145,29 @@ const JoblistingDetail = ({
           )}
 
           {(canEdit || canDelete) && (
-            <ul style={{ marginTop: '10px' }}>
-              <li>
-                <strong>Admin</strong>
-              </li>
+            <Flex column gap={7} style={{ marginTop: '10px' }}>
+              <h3>Admin</h3>
               {canEdit && (
-                <li>
-                  <Link to={`/joblistings/${joblisting.id}/edit`}>Rediger</Link>
-                </li>
+                <Link to={`/joblistings/${joblisting.id}/edit`}>
+                  <Button>
+                    <Icon name="create-outline" size={19} />
+                    Rediger
+                  </Button>
+                </Link>
               )}
               {canDelete && (
-                <li>
-                  <ConfirmModalWithParent
-                    title="Slett jobbannonse"
-                    message="Er du sikker på at du vil slette denne jobbannonsen?"
-                    onConfirm={onDeleteJoblisting}
-                  >
-                    <span className={styles.deleteButton}>Slett</span>
-                  </ConfirmModalWithParent>
-                </li>
+                <ConfirmModalWithParent
```

Comment:
Not sure if you've visited this component, but the confirm modal here feels kinda flat. Might benefit from a `danger` on the negative confirmation part. Same goes for the other places where that same component is used, at least those that are touched in this PR.

Reply:
Good point! I've considered it before, but forgot about it now. I made the confirm button to be `danger` by default.